examples/examples-release-15/src/main/java/io/kubernetes/client/examples/DeployRolloutRestartExample.java

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package io.kubernetes.client.examples;
+
+import java.io.IOException;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.util.Collections;
+
+import io.kubernetes.client.custom.V1Patch;
+import io.kubernetes.client.openapi.ApiClient;
+import io.kubernetes.client.openapi.ApiException;
+import io.kubernetes.client.openapi.Configuration;
+import io.kubernetes.client.openapi.apis.AppsV1Api;
+import io.kubernetes.client.openapi.models.V1Container;
+import io.kubernetes.client.openapi.models.V1Deployment;
+import io.kubernetes.client.openapi.models.V1DeploymentBuilder;
+import io.kubernetes.client.openapi.models.V1DeploymentSpec;
+import io.kubernetes.client.openapi.models.V1LabelSelector;
+import io.kubernetes.client.openapi.models.V1ObjectMeta;
+import io.kubernetes.client.openapi.models.V1PodSpec;
+import io.kubernetes.client.openapi.models.V1PodTemplateSpec;
+import io.kubernetes.client.util.Config;
+import io.kubernetes.client.util.PatchUtils;
+import io.kubernetes.client.util.wait.Wait;
+
+public class DeployRolloutRestartExample {
+	public static void main(String[] args) throws IOException, ApiException {
+		ApiClient client = Config.defaultClient();
+		Configuration.setDefaultApiClient(client);
+		AppsV1Api appsV1Api = new AppsV1Api(client);
+
+		String deploymentName = "example-nginx";
+		String imageName = "nginx:latest";
```

Comment:
fwiw, using the `latest` tag is an anti-pattern maybe make this a specific semantic version?

Reply:
👍🏼 Thanks for catching that! I applied a specific tag `1.21.6` instead of `latest`.